cmd/cluster-kube-apiserver-operator/main.go

```diff
@@ -1,32 +1,30 @@
 package main
 
 import (
-	goflag "flag"
 	"fmt"
 	"math/rand"
 	"os"
 	"time"
 
 	"github.com/spf13/cobra"
-	"github.com/spf13/pflag"
-
-	utilflag "k8s.io/apiserver/pkg/util/flag"
 	"k8s.io/apiserver/pkg/util/logs"
 
 	"github.com/openshift/cluster-kube-apiserver-operator/cmd/cluster-kube-apiserver-operator/render"
 	"github.com/openshift/cluster-kube-apiserver-operator/pkg/cmd/operator"
 	"github.com/openshift/library-go/pkg/operator/staticpod/installerpod"
+	"github.com/openshift/library-go/pkg/serviceability"
 )
 
 func main() {
 	rand.Seed(time.Now().UTC().UnixNano())
 
-	pflag.CommandLine.SetNormalizeFunc(utilflag.WordSepNormalizeFunc)
-	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
```

Comment:
@deads2k i think you copied this from `kubectl` but for some reason it brings all test flags into this command... I searched trough imports and haven't found any offending package that does that so far. 

Not sure whether we want to risk that in future some of the vendored packages pollute the go flag and we get "extra" flags set?

/cc @soltysh